src/fetcher.js

```diff
@@ -1290,13 +1296,32 @@ class Fetcher {
     }
   }
 
+  /* refresh  Reload data from a given document
+  **
+  ** @param  {NamedNode} term -  An RDF Named Node for the eodcument in question
+  ** @param  {function } userCallback - A function userCallback(ok, message, response)
+  */
   refresh (term, userCallback) { // sources_refresh
     this.fireCallbacks('refresh', arguments)
 
     this.nowOrWhenFetched(term, { force: true, clearPreviousData: true },
       userCallback)
   }
 
+ /* refreshIfExpired   Conditional refresh if Expired
+ **
+ ** @param  {NamedNode} term -  An RDF Named Node for the eodcument in question
+ ** @param  {function } userCallback - A function userCallback(ok, message, response)
+ */
+  refreshIfExpired (term, userCallback) {
+    let exp = this.getHeader(term, 'Expires')
+    if (!exp || (new Date(exp).getTime()) <= (new Date().getTime())) {
```

Comment:
Note for the future: `new Date().getTime() === Date.now()`

Reply:
Meaning?   NO, we are not comparing equal we are comparing one greater than the other.    don't understand that comment.  The above is the stackoverflow recommended way of compating dates

Reply:
I meant that `(new Date().getTime())` is equivalent to `Date.now()`, so in the future you can just write `new Date(exp).getTime() < Date.now()`.

Reply:
Ah I see